rosdep/python.yaml

```diff
@@ -2812,6 +2812,10 @@ python-qt5-bindings-gl:
     xenial: [python-pyqt5.qtopengl]
     yakkety: [python-pyqt5.qtopengl]
     zesty: [python-pyqt5.qtopengl]
+python-qt5-bindings-qsci:
+  debian: [python-pyqt5.qsci]
```

Comment:
This package is not available on `jessie`, we should provide either a pip rule or a `null` rule for Debian Jessie

Reply:
I think we can ignore that for jessie as it's EOL'd today.